ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/validation/CommitterModelConformanceValidator.scala

```diff
@@ -108,19 +112,22 @@ private[transaction] class ModelConformanceValidator(engine: Engine, metrics: Me
       } yield ()
       stepResult.fold(identity, _ => StepContinue(transactionEntry))
     } catch {
+      // Missing package or contract not specified as input
```

Comment:
Should we be treating packages and contracts the same here? Afaict a missing package really is an issue in the participant right now whereas a missing contract can arise from committer pruning.

Reply:
Discussed above